example_app_generator/generate_stuff.rb

```diff
@@ -89,10 +89,12 @@ def using_source_path(path)
 
 generate('model thing name:string')
 generate('helper things')
-generate('scaffold widget name:string category:string instock:boolean foo_id:integer bar_id:integer --force')
+generate('scaffold product name:string category:string instock:boolean foo_id:integer bar_id:integer --force')
```

Comment:
At this point the project stops being fun 😠 

Apparently there other tests assumed that the `Widget` constant is created and I cannot figure how to make the testing of IntegrationGenerator and RequestGenerator work, sometimes the `<%= index_helper %>_path` works, sometimes or the `_url` helper.

I still think that generated request specs is important and very useful but for working on this **as a lone wolf I am out**. So please help, take over oder schedule chatting/pair programming (personally; I am more available in February).

---

Log of the smoke tests:

```
Failures:

  1) Using rspec-mocks with models supports stubbing attribute methods on models
     Failure/Error: a_widget = Widget.new
     
     NameError:
       uninitialized constant Widget
     # ./spec/features/model_mocks_integration_spec.rb:10:in `block (2 levels) in <top (required)>'

  2) Using rspec-mocks with models supports stubbing class methods on models
     Failure/Error: allow(Widget).to receive(:all).and_return(:any_stub)
     
     NameError:
       uninitialized constant Widget
     # ./spec/features/model_mocks_integration_spec.rb:5:in `block (2 levels) in <top (required)>'

  3) Widgets GET /widgets works! (now write some real specs)
     Failure/Error: get widgets_index_path
     
     NameError:
       undefined local variable or method `widgets_index_path' for #<RSpec::ExampleGroups::Widgets::GETWidgets:0x00007ff1f27043d8>
     # ./spec/requests/widgets_spec.rb:6:in `block (3 levels) in <top (required)>'
```

Reply:
Have you had a chance to take another look? I can lend a hand.

Reply:
Yeah, I'll write you an email to chat up or something